src/fighter_state.rs

```diff
@@ -971,6 +971,7 @@ fn throwing(
                 ItemKind::Health {
                     health: item_health,
                 } => {
+                    // Obs: It should not happen
                     // Refill player's health
                     **health = (**health + item_health).clamp(0, stats.max_health);
```

Comment:
```suggestion
                    panic!("Health items should be used immediately, and can't be thrown");
```

I think it'd be better to take out the health logic here and panic if this happens to indicate that this shouldn't happen.

Reply:
You are completely right. I don't know how i haven't thought on it.